internal/pkg/cli/pipeline_show.go

```diff
@@ -131,26 +140,58 @@ func (o *showPipelineOpts) askPipelineName() error {
 		return nil
 	}
 
+	// return pipelineName from manifest if found
 	pipelineName, err := o.getPipelineNameFromManifest()
+	if err == nil {
+		o.pipelineName = pipelineName
+		return nil
+	}
+
+	if !errors.Is(err, workspace.ErrNoPipelineInWorkspace) {
```

Comment:
Should we return an error if error type is not "ErrNoPipelineInWorkspace" and log "No pipeline manifest in ..." when error type is "ErrNoPipelineInWorkspace"?

Reply:
@efekarakus had recommended that we don't return an error -- this is similar to the behavior to the behavior in [app_show](https://github.com/aws/amazon-ecs-cli-v2/blob/master/internal/pkg/cli/app_show.go#L234-L236) for no apps found.